lib/src/utils/previous_search_result_intent.dart

```diff
@@ -0,0 +1,9 @@
+import 'package:flutter/material.dart';
+
+class PreviousSearchResultIntent extends Intent {
+  late final int previousIndex;
+
+  PreviousSearchResultIntent(int currentIndex) {
```

Comment:
`const` ?

Reply:
Can't have const when the variables are marked as `late`

Reply:
ok